test/runtests.jl

```diff
@@ -109,7 +109,7 @@ end
     filepath = get_example("capitol.tif")
     img = TiffImages.load(filepath)
     @test size(img) == (378, 504)
-    @test eltype(img) == Gray{Bool}
+    @test eltype(img) == Gray{N7f1}
```

Comment:
Why is the type changing here? Bilevel images should be a boolean.

Reply:
Although it's been a couple months since I wrote most of this, I suppose it's easier to avoid special-casing for one particular bit depth. Interpreting sample values as `true` or `false` doesn't make sense to me. `Gray{Bool}` and `Gray{N7f1}` both use one byte

Incidentally, this PR also addresses this error for compressed bi-level images:
```julia
ERROR: Strided bilevel TIFFs are not yet supported. Please open an issue against TiffImages.jl.
```